hugo/content/docs/metrics.md

```diff
@@ -4,196 +4,33 @@ title: Metrics
-The metric histogram will get the observation, where the time is measured in seconds.
-
-#### Benefits
-
-#### Tests
-
-In your application, you'd need to configure the metrics instance with a proper provider, so you get your application properly measured.
-
-However, you might want to avoid this in your test project. There, you can use the `NoMetrics` configuration provider. When instantiating the `Metrics` instance without any provider supplied, the `NoMetrics` provider will be used.
-
-The `Metrics.Instance` static member will implicitly create an instance of the `Metrics` class with `NoMetrics` provider.
-
-```csharp
-public class MyTestFixture {
-    static MyTextFixture() => MyAwesomeAppMetrics.Configure(Metrics.Instance);
-
-    // here are my other setups
-}
-```
-
-#### Environment-specific
-
-Another scenario would be that your organisation is using Datadog APM to measure apps in production, but when running locally, you'd still like to measure and you don't have the Datadog agent running on your dev machine.
-
-In this case, you can run Prometheus and Grafana locally in Docker and use different configuration providers, based on the environment name.
-
-```csharp
-public static class Measurements {
-    public static void ConfigureMetrics(string environment) {
-        IMetricsProvider configProvider = environment == "Development"
-            ? new PrometheusConfigurator()
-            : new StatsdConfigurator(new [] {
-                  new Label("app", "myAwesomeApp"),
-                  new Label("environment", environment)
-              });
-        );
-        MyAwesomeAppMetrics.Configure(Metrics.CreateUsing(configProvider));
-    }
-}
-```
-
-#### Libraries
-
-It's probably not the best idea for a library author to couple measurements to a specific product.
-
-Using this library, you can be vendor-agnostic when collecting the metrics and give your users the ability to use the provider they want.
-
-### Vendors support
-
-Currently, the library supports exposing metrics for:
-
-- Prometheus (using [prometheus-net](https://github.com/prometheus-net/prometheus-net))
-- Datadog Statsd (using [Dogstatsd](https://github.com/DataDog/dogstatsd-csharp-client))
-
-In addition, we provide a few "fake" configurators, which either do nothing, or allow you to look at the measurements locally.
-
-- NoMetrics: very fast, good for libraries as it's the default one
-- Microsoft Logging: logs measurements using Microsoft logging framework
-- InMemory: WIP
-
-## Built in metrics in Proto.Actor
-
-List all the metric names and based on module:
-https://github.com/asynkron/protoactor-dotnet/issues/948
+Proto.Actor is using metrics provided by [OpenTelemetry](https://opentelemetry.io/docs/reference/specification/metrics/). With simple Proto.Actor instrumentation from `Proto.OpenTelemetry`, it is possible to get insights how actor system is performing.
+Below it is possible to find more details regarding built in metrics.
```

Comment:
Mention here, that when using prometheus exporter directly in the application, the metric names get postfixed with unit name (e.g. `_seconds`).